src/i18n.js

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import Backend from 'i18next-http-backend';
+
+i18n
+    // load translation using http -> see /public/locales (i.e. https://github.com/i18next/react-i18next/tree/master/example/react/public/locales)
```

Comment:
Not sure how link https://github.com/i18next/react-i18next/tree/master/example/react/public/locales is relevant ... if u feel that it is needed to include some link i would put rather a guide according to which you created it.

Reply:
This could also be problematic if the tool was to run on an internal network.

Reply:
removed.